src/Hl7.Fhir.R5.Core/Model/Questionnaire.cs

```diff
@@ -49,8 +49,8 @@ internal string DebuggerDisplay
                     StringBuilder sb = new StringBuilder();
                     if (!string.IsNullOrEmpty(this.LinkId))
                         sb.AppendFormat(" LinkId=\"{0}\"", LinkId);
-                    if (!string.IsNullOrEmpty(this.Text?.Value))
-                        sb.AppendFormat(" Text=\"{0}\"", Text.Value);
+                    if (this.Text is not null)
```

Comment:
Yes, but `Questionnaire.Text` has different types for certain FHIR versions. First it was `string`, then it is `Markdown`. Because this file is used for R4, R4B and R5, this new line works now  for all of the FHIR versions.